docs/guide/docs/main/overview.md

```diff
@@ -1,74 +1,109 @@
+### Dumb Bots
 
-While you can develop both kind of bots with Botpress, you will really benefit most of it if you're trying to build a smart bot.
+A bot can be **dumb** meaning it doesn't understand much about conversations). Such chatbots are rules-based and usually rely heavily on pre-populated responses (e.g. buttons) or one-word answers. The data which they receive from the user rarely influences responses and the bot simply moves to the next step after a response.
+
+The beauty of this type of bot is that it is cheaper, easy to implement, 99% effective and solves issues fast. This is because they are built with a specific workflow in mind and once a user understands what the goal of the bot is, s/he can easily use the bot for this goal.
+
+### Smart Bots
+
+The description **smart** means that the bot can understand language and communicate in a human-like way. Such bots can extract useful information from conversations like dates, amounts, and locations.
+
+Such bots are indispensable as they can harvest additional data from the user. They can be used to broach abstract concepts such as gauging the sentiment and mood of the user. They can also help the streamlining of conversations by harvesting as much information as is possible from the user statement, making conversations shorter and more concise.
+
+While you can develop both kinds of bots with Botpress, you will benefit most if you're trying to build a smart bot.
 
 ## Anatomy of a bot
 
 A bot is pretty simple:
 
 1. It receives messages from Messaging **Channels**
 2. It **processes** these messages to understand, translate or escalate them
-3. It **decides** on what to respond back to the user
+3. It **decides** on what to respond to the user
 
 #### Lifecycle
 
 ![High-Level Diagram](assets/hld.png)
 
-<br/>
 
-In red are the developer extension points, i.e. where you will be able to inject code to customize the bot.
 
-## Channels
 
-A channel is a module that allows your bot to send and receive messages from a specific chat platform (Facebook Messenger, Telegram, etc.).
+The red dots are the developer extension points. These indicate where code can be injected to customize the bot.
+
+### Channels
 
-> Note: Unlike most other bot platforms, channels are installed and configured individually and locally, which means you have full control over the data that is transmitted between your bot and the chat platforms. Botpress does not proxy the messages to a 3rd party service as Microsoft Bot Framework does.
+A channel is a module that allows your bot to send and receive messages from a specific chat platform (Your website, Facebook Messenger, Telegram, etc.).
+
+> In Botpress, channels are installed and configured individually and locally, giving you full control over the data that is transmitted between your bot and the chat platforms. Botpress does not proxy the messages to a 3rd party service as Microsoft Bot Framework does.
```

Comment:
Do we really want to state the name of a competitor in our documentation?

Reply:
I would also remove that statement